EDU-471 cloud docs refresh branch (#2213)
temporalio/documentation: docs-src/cloud/pricing-support.md

---
id: pricing-support
title: What kind of support do I get with Temporal Cloud?
sidebar_label: Support

> description: With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects.

Reviewer: ```suggestion
description: A subscription to Temporal Cloud grants access to our support team of developer success engineers and solution architects.
```

tags:
  - temporal cloud
  - pricing
  - support
  - explanation
---

> With a subscription to Temporal Cloud, you also gain access to our support organization of developer success engineers and solution architects. Our experts assist with a range of work streams from Workflow design reviews to setting up observability to break/fix support with an agreed-upon set of SLAs.

Author: it is also apparently a commonly used term for this type of support: https://en.wikipedia.org/wiki/Break/fix

> The members of the Temporal Developer Success team are engineers who are Temporal experts and open source contributors. They're backed by the main contributors to the open source Temporal project. They provide deep knowledge of how Temporal works and how you can optimize your deployment.

Reviewer: ```suggestion
The Temporal Developer Success team can provide deep knowledge of how Temporal works and how you can optimize your deployment. This team is comprised of engineers who know Temporal inside and out, and continue to contribute to its development.
```
